frontend/javascripts/oxalis/geometries/plane.js

```diff
@@ -148,14 +148,13 @@ class Plane {
     );
   };
 
-  setPosition = (posVec: typeof THREE.Vector3): void => {
-    this.TDViewBorders.position.copy(posVec);
-    this.crosshair[0].position.copy(posVec);
-    this.crosshair[1].position.copy(posVec);
-    this.plane.position.copy(posVec);
-
-    const globalPosition = getPosition(Store.getState().flycam);
-    this.plane.material.setGlobalPosition(globalPosition);
```

Comment:
This is kind of weird, why wasn't `posVec` given as the argument here, before?!

Reply:
Ah, dang. I checked the caller in `SceneController.update()` where it should not matter. However, there's also `SceneController.updateSceneForCam()` where the third dimension is offset so that the skeleton is before the plane. So, there can be a difference.

Do you know what the difference between `update` and `updateSceneForCam` is? I'm a bit confused about their purposes :thinking: Happy to have a call about this in case it's easier :see_no_evil:

Reply:
To me it looks like the `update` method is used to update the scale and position of planes if the user moved/zoomed. The `updateSceneForCam` method is used during rendering of the planes, where if plane A is rendered, planes B and C are hidden to avoid artifacts (plane A is also offset to allow to see the skeleton tracing behind the plane).

Reply:
Hmm, at least for the position, it's not clear to me why the two update mechanisms are necessary. In the end, updating the position in the plane material should only be relevant for rendering (if it's not rendered, it shouldn't matter). So, only `updateSceneForCam`'s `setPosition` call seems crucial to me. Or am I missing something?